src/libs/sensors-logging.ts

```diff
@@ -121,12 +215,17 @@ class DataLogger {
         [DatalogVariable.longitude]: { value: `${vehicleStore.coordinates.longitude?.toFixed(6)} °` || 'Unknown', ...timeNowObj },
       }
       /* eslint-enable vue/max-len, prettier/prettier, max-len */
+      const veryGenericData = this.collectVeryGenericData(timeNowObj)
+      variablesData = { ...variablesData, ...veryGenericData }
 
-      this.currentCockpitLog.push({
+      const logPoint: CockpitStandardLogPoint = {
```

Comment:
This change can also be accomplished with `this.currentCockpitLog.push({...} as CockpitStandardLogPoint)`.

No need to change, just a tip.